src/homepageExperience/containers/CliWizard.test.tsx

```diff
@@ -0,0 +1,36 @@
+// Libraries
+import React from 'react'
+import {fireEvent, screen} from '@testing-library/react'
+import {renderWithRedux} from 'src/mockState'
+
+// Components
+import {CliWizard} from 'src/homepageExperience/containers/CliWizard'
+
+jest.mock('canvas-confetti')
+jest.mock('src/homepageExperience/assets/sample.csv', () => {
+  return 'csv, csv, csv'
+})
+jest.mock('assets/images/sample-csv.png', () => {
+  return 'csv screenshot'
+})
+
+const setup = () => {
+  return renderWithRedux(<CliWizard />)
+}
+
+describe('Navigation', () => {
+  describe('Next and Previous Buttons', () => {
+    it('cannot click next on final step', async () => {
+      setup()
+      await fireEvent.click(screen.getByText('Finished!'))
+      const nextButton = screen.getByTestId('cli-next-button')
+      expect(nextButton).toHaveAttribute('disabled')
+    })
+    it('cannot click previous on first step', async () => {
+      setup()
+      await fireEvent.click(screen.getByText('Overview'))
+      const prevButton = screen.getByTestId('cli-prev-button')
+      expect(prevButton).toHaveAttribute('disabled')
+    })
```

Comment:
I think it'd be valuable to add a couple more tests here that test that clicking `Next` and `Previous` work as expected